production/helm/loki/templates/monitoring/metrics-instance.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.monitoring.serviceMonitor.enabled }}
+{{- with .Values.monitoring.serviceMonitor.metricsInstance }}
```

Comment:
Is `metricsInstance` is `null` this will be empty, won't it?

Reply:
yes, I believe so, `null` or an empty object. But you can test via the `helm template` command